Podfile

```diff
@@ -154,7 +154,7 @@ target 'WordPress' do
     ## Third party libraries
     ## =====================
     ##
-    pod '1PasswordExtension', '1.8.5'
+    pod '1PasswordExtension', '1.8.6'
```

Comment:
Just a small note for when you merge the latest from develop:

You'll notice this dependency is gone.  That's because WPiOS doesn't directly depend on 1Password and only uses it through WPAuthenticator.  Since WPAuthenticator already defines the dependency,  it doesn't need to be here.

Reply:
ah, nice! 🎉